src/node/create-webpack-config-static.js

```diff
@@ -12,7 +12,16 @@ const createWebpackConfigBase = require('./create-webpack-config-base');
 function createWebpackConfigStatic(
   batfishConfig: BatfishConfiguration
 ): Promise<webpack$Configuration> {
-  return createWebpackConfigBase(batfishConfig).then(baseConfig => {
+  // For the static build always set up Babel to compile only what's necessary
+  // for the current version of Node.
+  return createWebpackConfigBase(
+    Object.assign({}, batfishConfig, {
+      babelPresetEnvOptions: {
+        useBuiltIns: true,
+        targets: { node: 'current' }
```

Comment:
I don't understand why we only need to compile what's necessary for the current version of Node. Can you explain why this is the case?

Reply:
When we're using Node 6, we get a lot of ES2015 without compilation. So we should in theory have faster compile-time and maybe even faster runtime if, for example, we don't compile classes and arrow functions, which Node 6 understands.

Reply:
So the set of ES features that get compiled, and end up in the static bundle, are driven by the version of Node that compiles the assets? If I'm understanding this correctly, I feel like this could result in some confusion.

Reply:
As we clarified in chat: The misunderstanding here was about where and when the static bundle runs. The output of this Webpack task is just `static-render-pages.js`, and that only runs in Node.